draft-ietf-quic-recovery.md

```diff
@@ -338,7 +338,7 @@ mean variation.
 The calculation of smoothed_rtt uses path latency after adjusting RTT samples
 for acknowledgement delays. These delays are computed using the ACK Delay
 field of the ACK frame as described in Section 19.3 of {{QUIC-TRANSPORT}}.
-For packets sent in the ApplicationData packet number space, a peer limits
+For packets sent in the Application Data packet number space, a peer limits
```

Comment:
This is two separate words in the other drafts; I changed it in the prose but left it in the pseudo-code, since it's an enum value there.